src/pages/billing/Project.js

```diff
@@ -258,32 +335,27 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
     setProjectUsers(projectUsers)
   })
 
-
   // Lifecycle
-  Utils.useOnMount(() => {
-    refresh()
-    loadBillingInfo()
-  })
+  Utils.useOnMount(refresh)
```

Comment:
Because React will throw a warning if an effect's function returns anything, as a rule we wrap them:
```suggestion
  Utils.useOnMount(() => { refresh() })
```